client/block.c

```diff
@@ -50,6 +50,7 @@ struct orphan_block;
 struct block_internal {
 	struct ldus_rbtree node;
 	xdag_hash_t hash;
+	xdag_remark remark;
```

Comment:
this will waste a lot of ram/tmpfile... we need to keep remark in ram/tmpfile for every block ?
I think it could be read from storage on purpose.